Userland/Utilities/diff.cpp

```diff
@@ -37,11 +37,29 @@ int main(int argc, char** argv)
 
     auto hunks = Diff::from_text(file1->read_all(), file2->read_all());
     for (const auto& hunk : hunks) {
-        outln("Hunk: {}, {}", hunk.original_start_line, hunk.target_start_line);
-        for (const auto& line : hunk.removed_lines) {
+        auto original_start = hunk.original_start_line;
+        auto target_start = hunk.target_start_line;
+        auto num_added = hunk.added_lines.size();
+        auto num_removed = hunk.removed_lines.size();
+
+        StringBuilder sb;
+        // Source line(s)
+        sb.appendff("{}", original_start);
+        if (num_removed > 1)
+            sb.appendff(",{}", original_start + num_removed - 1);
+
+        // Action
+        sb.append(num_added > 0 && num_removed > 0 ? "c" : num_added > 0 ? "a"
```

Comment:
Nested ternaries are evil! :P
Please use `if`s here, especially if it doesn't comfortably fit in a line anyway.